oracle/learning-library
# cleanup for workshop microservices with converged database #442 per QA check

QA Check for DB World:

  In Introduction Lab: 

        Rename the title Build microservices with converged Oracle database to Introduction
        Rename the heading 2 Introduction and Overview to About this Workshop
        Before the “You may now proceed to the next lab.” add heading 3 Objectives section that covers the objectives of this workshop
        After the Objectives, add another heading 3 Prerequisites section

    Please make sure that all images contain alternate text for accessibility in the introduction lab and in labs -1,2,3,4,5,6,7
    In Lab 1,

        Fix how the title is displayed in the lab
        Number the instructions in task 1,2,3,8,9
        In task 7, fix how the note is displaying on GitHub.io pages Use the format > **Note:** for displaying note.

    In Lab 2 - Number the instructions in task 2, 9, 11
    Add “You may now proceed to the next lab.” at the end of the last task in labs 1,2,3,4,6,7
    Check and update the task titles to initial capitalization.
    In odbw-freetier folder - manifest.json file rename Getting Started to Get Started
    In odbw-livelabs folder - manifest.json file rename Getting Started to Get Started and move it after the Introduction lab
